.github/actions/javascript/authorChecklist/index.js

```diff
@@ -49929,8 +49929,7 @@ Object.defineProperty(exports, "__esModule", ({
 }));
```

Comment:
Just to confirm that the changes in this file is necessary for RN upgrade

Reply:
Yeah, I think that might be expected. Sometimes the output of ncc changes when we update dependency versions.